dependencies.yml

```diff
@@ -90,7 +90,7 @@ io.prometheus:
   simpleclient_common: { version: '0.1.0' }
 
 io.zipkin.brave:
-  brave: { version: &BRAVE_VERSION '4.6.0' }
+  brave: { version: &BRAVE_VERSION '4.9.0' }
```

Comment:
ps you can remove the transitive dep on io.zipkin.java:zipkin (as you aren't using it)

Reply:
Hmm - we are using `zipkin.Constants` / `zipkin.TraceKeys`. Is there a replacement for those in `zipkin2` or should we use something like this to ensure compatibility?

https://github.com/opentracing/opentracing-java/blob/master/opentracing-api/src/main/java/io/opentracing/tag/Tags.java